Sofa/Component/MechanicalLoad/src/sofa/component/mechanicalload/OscillatingTorsionPressureForceField.h

```diff
@@ -127,13 +127,13 @@ protected :
     Coord getVecFromRotAxis( const Coord &x );
     Real getAngle( const Coord &v1, const Coord &v2 );
 
-    std::vector<Real> relMomentToApply;   // estimated share of moment to apply to each point
-    std::vector<bool> pointActive;        // true if moment is applied to specific point (surface)
-    std::vector<Coord> vecFromCenter;     // vector from rotation axis for all points
-    std::vector<Real> distFromCenter;     // norm of vecFromCenter
-    std::vector<Coord> momentDir;         // direction in which to apply a moment
-    std::vector<Coord> origVecFromCenter; // vector from rotation axis for all points in original state
-    std::vector<Coord> origCenter;        // center of rotation for original points
+    sofa::type::vector<Real> relMomentToApply;   // estimated share of moment to apply to each point
+    sofa::type::vector<bool> pointActive;        // true if moment is applied to specific point (surface)
+    sofa::type::vector<Coord> vecFromCenter;     // vector from rotation axis for all points
+    sofa::type::vector<Real> distFromCenter;     // norm of vecFromCenter
+    sofa::type::vector<Coord> momentDir;         // direction in which to apply a moment
+    sofa::type::vector<Coord> origVecFromCenter; // vector from rotation axis for all points in original state
+    sofa::type::vector<Coord> origCenter;        // center of rotation for original points
```

Comment:
You are changing the type so this PR is "breaking" ℹ️

Reply:
I agree. I thought I set it, but no. Thanks